### PR TITLE
Add i18n

### PR DESCRIPTION
**Pull request by hoang-rio**
Add i18n and translate client dashboard to Vietnamese

### CODE REVIEW

**Comment by ameshkov** [comment]
Uh, that's a huge PR, thank you!

Tbh, we wanted to finalize the dashboard UI first, and add translations a bit later after that. We will reuse this PR partly when it's time: https://github.com/AdguardTeam/AdGuardHome/issues/442

**Comment by ameshkov** [comment]
I see you're supporting this PR, thank you for that!

In order for this PR to be merged, there's a few changes that need to be done:

1. Base language (English) strings need to be added as a separate file: `client/src/__locales/en.js`
2. Every string should have an identifier, e.g. `Blocked malware/phishing` -> `stat_blocked_malware`
3. In the JS files, we should use these identifiers instead of plain English strings

The reason for this change is translation automation via adguard.oneskyapp.com or crowdin. If we don't do it and keep using the current approach, every change to the base language will basically add a new string for translation instead of changing the existing one.

**Comment by hoang-rio** [comment]
> I see you're supporting this PR, thank you for that!
> 
> In order for this PR to be merged, there's a few changes that need to be done:
> 
> 1. Base language (English) strings need to be added as a separate file: `client/src/__locales/en.js`
> 2. Every string should have an identifier, e.g. `Blocked malware/phishing` -> `stat_blocked_malware`
> 3. In the JS files, we should use these identifiers instead of plain English strings
> 
> The reason for this change is translation automation via adguard.oneskyapp.com or crowdin. If we don't do it and keep using the current approach, every change to the base language will basically add a new string for translation instead of changing the existing one.

Ok. I will update it.

File: client/src/__locales/en.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        example_meaning_host_block: 'AdGuard Home will now return 127.0.0.1 address for the example.org domain (but not its subdomains).',
+        example_comment: '! Here goes a comment',
+        example_comment_meaning: 'just a comment',
+        example_comment_hash: '# Also a comment',
+        all_filters_up_to_date_toast: 'All filters are already up-to-date',
+        updated_upstream_dns_toast: 'Updated the upstream DNS servers',
+        dns_test_ok_toast: 'Specified DNS servers are working correctly',
+        dns_test_not_ok_toast: 'Server "{{key}}": could not be used, please check that you\'ve written it correctly',
+        // Logs
+        ublock_btn: 'Unblock',
```
Comment (ameshkov) [inline client/src/__locales/en.js]: `ublock_btn` -> `unblock_btn`

**Comment by ameshkov** [comment]
@IldarKamalov please review and also check out this task: https://github.com/AdguardTeam/AdGuardHome/issues/442#issuecomment-437709171

**Pull request merged**